evennia/server/is_typing.py

```diff
@@ -0,0 +1,71 @@
+"""
+This module allows users based on a given condition (defaults to same location) to see
+whether applicable users are typing or not. Currently, only the webclient is supported.
+"""
+
```

Comment:
If the user wants to customize this functionality, they need to replace this module in `settings.INPUT_FUNC_MODULES`, implementing both of `is_typing_setup` and `is_typing_state`. Exactly what is required for this customization should be documented here (and added to documentation as well, but that can come later).